go/bind/core/node.go

```diff
@@ -38,6 +38,18 @@ type Node struct {
 }
 
 func NewNode(r *Repo, driver ProximityDriver) (*Node, error) {
+	var dialer net.Dialer
+	net.DefaultResolver = &net.Resolver{
+		PreferGo: false,
+		Dial: func(context context.Context, _, _ string) (net.Conn, error) {
+			conn, err := dialer.DialContext(context, "udp", "1.1.1.1:53")
```

Comment:
Can you use the same DNS address as here? https://github.com/ipfs-shipyard/gomobile-ipfs/blob/9574047413bb2bf4446e8ae105fa11ddd35e934a/go/bind/core/dnsconfig.go#L79

Reply:
I pushed the change. We only use one address. Is that OK?

Reply:
Ok if we can't set a secondary address 👍

Reply:
I looked at the golang source code to see if DialContext allows multiple addresses (maybe separated by comma), but it does not.
https://github.com/golang/go/blob/master/src/net/dial.go#L213

Merging with only the primary address.